includes/rewrites.php

```diff
@@ -36,6 +36,7 @@ function rewrite_entity_descr($descr)
     $descr = str_replace('PMOD', 'PSU', $descr);
     $descr = preg_replace('/^temperatures /', '', $descr);
     $descr = preg_replace('/^voltages /', '', $descr);
+    $descr = str_replace('PowerSupply', 'PSU ', $descr);
```

Comment:
should there really be a space after PSU?

Reply:
Yes. If you check the before screenshots, old name was PowerSupplyX (X being the PSU module) so if there isnt a space, it would look PSU1 and PSU2.